src/storage/mvcc/reader/point_getter.rs

```diff
@@ -239,9 +257,7 @@ impl<S: Snapshot> PointGetter<S> {
             use_near_seek = true;
 
             let cursor_key = self.write_cursor.key(&mut self.statistics.write);
-            if !Key::is_user_key_eq(cursor_key, user_key.as_encoded().as_slice()) {
-                return Ok(None);
-            }
+            // No need to compare user key because it uses prefix seek.
```

Comment:
Seems they are leftovers after `PointGetter` changed to always use prefix seek. Now the "multi" option looks totally useless to me 😂

Reply:
Yes. I kept the `multi` option to tell the `cursor` use `ScanMode::Mixed` but seems point getter can always use `ScanMode::Mixed`. Let me remove it in another PR.